java/ql/src/Telemetry/AutomodelExtractCandidates.ql

```diff
@@ -0,0 +1,39 @@
+/**
+ * Surfaces the endpoints that are not already known to be sinks, and are therefore used as candidates for
+ * classification with an ML model.
+ *
+ * Note: This query does not actually classify the endpoints using the model.
+ *
+ * @name Automodel candidates
+ * @description A query to extract automodel candidates.
+ * @kind problem
+ * @severity info
+ * @id java/ml-powered/extract-automodel-candidates
+ * @tags automodel extract candidates
```

Comment:
Make sure you add an `internal` tag to all these queries so we don't accidentally include them in the code scanning suites.

Reply:
Thank you for that catch. You mean like `@tags internal automodel extract candidates`? I'll add that tag, but I noticed that there's not one Java query that uses this tag currently, I only could find some csharp queries that do by searching for `tags.*internal` project-wide.